src/Coin.js

```diff
@@ -0,0 +1,77 @@
+const Coin = (props) => {
+
+    const handleClick = (e) => {
+        const siblingEl = e.target.nextSibling;
+        siblingEl.classList.toggle('show');
+    }
+
+    // ***** Conditional colour styling for price changes ****** //
+    const change24hStyle = props.change24h > 0 ? {color: 'green'} : {color: 'red'};
+    const change7dStyle = props.change7d > 0 ? {color: 'green'} : {color: 'red'};
+    const change30dStyle = props.change30d > 0 ? {color: 'green'} : {color: 'red'};
+    const athStyle = props.ath > 0 ? {color: 'green'} : {color: 'red'};
+    const atlStyle = props.atl > 0 ? {color: 'green'} : {color: 'red'};
+
+    
+    
+
+    return (
+        <li className="coin">
+            <div className="img-container">
+                <img src={props.image} alt="" />
```

Comment:
missing alt=''
could use {props.name}

Reply:
Thanks, added `<img src={props.image} alt={`Logo for ${props.name}`} />`